common/telemetry.js

```diff
@@ -43,5 +51,6 @@ export {
   publishWriteClient,
   networkInfoWriteClient,
   recordPublishTelemetry,
-  recordNetworkInfoTelemetry
+  recordNetworkInfoTelemetry,
+  batchMetricsWriteClient
```

Comment:
```suggestion
  recordNetworkInfoTelemetry
```

We won't need this one if we extend existing publish metric.